docs/OmniMIDI/Soundfonts/recommendations.md

```diff
@@ -6,4 +6,39 @@ Here are some great soundfonts which we recommend you to use for you endevaours
 
     These soundfonts were created by amazing individuals who worked hard to create them and are kind enough to offer them for free for you to use. Please show some love by crediting the creators if you make videos or use them!
 
-> TODO: im not sure what soundfonts would be good. someone pls take over
+## Arachno
+
+Arachno is a really nice soundfont, it's one of the most used soundfonts in the Black MIDI Conmunity.
```

Comment:
Should we add, "for GM sounds" at the end of this? Most of the time arachno is used for its instruments other than the default piano one. I would also like to make it clear that this should be accompanied by another soundfont like the ones you added here.

Reply:
Sure.